README.md

```diff
@@ -20,26 +20,28 @@ Sample Node service app that connects to a Laserfiche Cloud or Self-Hosted Repos
   - [EU Cloud](https://app.eu.laserfiche.com/laserfiche)
   - [US Cloud](https://app.laserfiche.com/laserfiche)
 
-- Using the app picker, go to the 'Account' page
-- Click on the 'Service Principals' tab
-- Click on the 'Add Service Principal' button to create an account to be used to run this sample service
-- Add access rights to the repository and click the 'Create' button
-- View the created service principal and click on the 'Create Service Principal Key(s)' button
-- Save the Service Principal Key for later use
+- Using the app picker, go to the 'Account' page.
+- Click on the 'Service Principals' tab.
+- Click on the 'Add Service Principal' button to create an account to be used to run this sample service.
+- Add access rights to the repository and click the 'Create' button.
+- View the created service principal and click on the 'Create Service Principal Key(s)' button.
+- Save the Service Principal Key for later use.
 
 ### 2. Create an OAuth Service App
 
 - Navigate to Laserfiche Developer Console:
   - [CA Cloud](https://app.laserfiche.ca/devconsole/)
   - [EU Cloud](https://app.eu.laserfiche.com/devconsole/)
   - [US Cloud](https://app.laserfiche.com/devconsole/)
-- Click on the 'New' button and choose the 'Create a new app' option
-- Select the 'Service' option, enter a name, and click the 'Create application' button
-- Select the app service account to be the one created on step 1 and click the 'Save changes' button
-- Click on the 'Authentication' Tab and create a new Access Key
-- Select the first option (i.e. Create a public 'Access Key'...)
-- Click the 'Download key as base-64 string' button for later use
-- Click OK
+- Click on the 'New' button and choose the 'Create a new app' option.
+- Select the 'Service' option, enter a name, and click the 'Create application' button.
+- Select the app service account to be the one created on step 1 and click the 'Save changes' button.
+- Click on the 'Authentication' Tab and create a new Access Key.
+- Select the first option (i.e. Create a public 'Access Key'...).
+- Click the 'Download key as base-64 string' button for later use.
+- Click OK.
+- Select the required scope(s) in the 'Authentication' tab. For running this sample project, both 'repository.Read' and 'repository.Write' scopes are required.
```

Comment:
could you also fix the sample code to use correct space delimited scope?
https://github.com/Laserfiche/lf-sample-repository-api-nodejs/blob/v1/index.ts#L42

Reply:
I fixed the issue.